core/util/src/main/scala/net/liftweb/util/Props.scala

```diff
@@ -85,6 +113,38 @@ object Props extends Logger {
   }
 
   /**
+   * Updates Props to find property values in the argument BEFORE looking in the standard Lift prop files.
+   * @param provider Arbitrary map of property key -> property value
+   */
+  def prependSource(provider:PropProvider):Unit = sources = provider :: sources
+
+  /**
+   * Updates Props to find property values in the argument AFTER first looking in the standard Lift prop files.
+   * @param provider Arbitrary map of property key -> property value
+   */
+  def appendSource(provider:PropProvider):Unit = sources = sources :+ provider
+
+  /**
+   * Removes any PropProvider sources which match the predicate
+   * @param removeIf predicate for selecting PropProviders to remove
+   */
+  def removeSources(removeIf: PropProvider=>Boolean):Unit =
+    sources = sources.filterNot(removeIf)
+
+  /**
+   * Updates Props to find values in the argument when interpolating values found in sources.
+   * @param provider Arbitrary map of property key -> property value
+   */
+  def appendInterpolator(provider:PropProvider):Unit = interpolators = interpolators :+ provider
+
+  /**
+   * Removes any PropProvider interpolators which match the predicate
+   * @param removeIf predicate for selecting PropProviders to remove
+   */
+  def removeInterpolators(removeIf: PropProvider=>Boolean):Unit =
+    interpolators = interpolators.filterNot(removeIf)
```

Comment:
Why not expose everything except for prepend/append as `updateInterpolators(updater: Seq[PropProvider])`? That way the user can do whatever they want.